build/BuildFunctions.psm1

```diff
@@ -53,77 +40,30 @@ function Initialize-PsTask([string]$task) {
     }
 }
 
-function Initialize-TsTask([string]$task) {
-    Write-Host "Initialize TypeScript task $task"
-    pushd $task
-    try {
-        if (-not (Test-Path node_modules)) {
-            npm install --loglevel error | Out-Null
-        }
-    } finally {
-        popd
-    }
-}
+function Invoke-AllMochaTests([string[]]$tasks, [string]$reporter, [switch]$throwOnError) {
+    Write-Host "Testing Tasks"
```

Comment:
Should we put `$tasks` in here so we know which ones we're testing?

Reply:
I don't think it is necessary. Each `Invoke-MochaTest` invocation says which test is being run already.